deployments/helm/README.md

```diff
@@ -0,0 +1,22 @@
+## Install KubeArmor using helm
+* Install kubearmor via helm, enable/disable kuberarmor relay by specifying either true or false.
+* Specify the namespace.
+* Specify environment depends on your environment like { docker, microk8s, minikube, k3s and generic (GKE, EKS)} by default it is generic.
+
+```
+helm upgrade --install kubearmor . \
+    --set kubearmorrelay.enabled=true \
+    --set environment.name=<environment> \
+    --set namespace.name=<namespace> \
+    -n <namespace>
+
+```
+Check if all the pods are up and running.
+```
+kubectl get all -n <namespace>
+```
+
+## To uninstall KubeArmor using helm
+```
+helm uninstall kubearmor -n <namespace>
```

Comment:
Namespace based uninstall didn't work for me during testing.
I installed in kaspace namespace and tried to uninstalled it said:
```
❯ helm uninstall kubearmor -n kaspace
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/rahul/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/rahul/.kube/config
Error: uninstall: Release not loaded: kubearmor: release: not found
```

Reply:
@nyrahul can u please send me the installation command which you have run?

Reply:
`helm upgrade --install kubearmor . --set kubearmorrelay.enable=true --set environment.name=k3s --set namespace.name=kaspace`

Reply:
Fine, use -n kaspace, because if we don't use helm release will be on default namespace, so I have given -n also in installation command.
if you want to uninstall now, please remove -n kaspace and run helm uninstall command

Reply:
I noticed that if I simply give `helm uninstall kubearmor` it uninstalls kubearmor regardless of the namespace.

Reply:
Yes, that is the exact behavior of helm, so to remove all confusions I have mentioned -n option in the commands